### PR TITLE
Label on assets

### PR DESCRIPTION
**Pull request by thomax**

### CODE REVIEW

File: src/assets/assetsClient.js [hunk 1]
```diff
@@ -21,13 +21,15 @@ assign(AssetsClient.prototype, {
       : {}
 
     const assetEndpointSegment = ASSET_TYPES_TO_ENDPOINT[assetType]
+    const query = options.label ? `?label=${options.label}` : ''
```
Comment (rexxars) [inline src/assets/assetsClient.js]: `encodeURIComponent(options.label)` is probably wise

Comment (rexxars) [inline src/assets/assetsClient.js]: Also, maybe use a label in the tests with something that would need encoding. A space or similar.

**Comment by bjoerge** [comment]
Just out of curiosity: what is this for?

**Comment by thomax** [comment]
@bjoerge assets upload route in the api does not allow the user to mint his/her own object its. Which is probably good. Thus, label can be used as alternative way of quickly deciding if the image you're about to upload already has been dealt with.

File: src/assets/assetsClient.js (hunk 1 above)
Comment (thomax) [inline src/assets/assetsClient.js]: Good point. Fixed!

**Pull request merged**